### ISSUE TITLE
Make a function returning the PNK at stake by a juror in a dispute

### ISSUE THREAD

**Issue by clesaege** [kleros/kleros#7]

**Comment by clesaege** [comment]
Note, it's at stake in a dispute, not the total at stake.

**Comment by clesaege** [comment]
From the PR I've seen, it seems that it's not possible to get it without giving the list first. So we may just need to abandon having this functionality.
What do you think @n1c01a5 @klassicd ?

**Comment by clesaege** [comment]
Or we can loop over, but put a huge warning that this should only be used by the interface as it would be O(n) and could blow the gas limit?

**Comment by epiqueras** [comment]
@clesaege 
We don't have to loop over disputes.
We just have to loop over draws for a given dispute or require the user to pass in `_draws`.

**Comment by klassicd** [comment]
I agree with @epiqueras, the method signature `(address _jurorAddress, uint _disputeID, uint[] _draws)` is the same one used for public methods `voteRuling`, `penalizeInactiveJuror` and `validDraws`. 

I would like to write a test, but could use some help getting it to pass due to the error `The contract code couldn't be stored, please check your gas amount.`.  Anyone else run into this before?  The tests stall when using Parity and throw this error with Ganache (first 4 tests pass).  

Also, when I migrate with truffle locally, it produces 12+ more files in `build/contracts` than what's in master.  Should I check in any of the new or modified build files to my PR?

**Comment by clesaege** [comment]
Requiring the user to pass the draw makes the method useless as the user would already know the value at stake if he knows the draws.
We can either abandon doing such method, or loop over the draws, making it O(n) which is ok if used for interface but not called by other contracts.
